### PR TITLE
Add repeats for MpTelemetry-2.0 JaxRs Tests

### PR DESCRIPTION
**Pull request by Yas-Franco**
For issue #28099

### CODE REVIEW

**Comment by Yas-Franco** [comment]
#build

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/jaxrspropagation/methods/JaxRsMethodTestServlet.java [hunk 1]
```diff
@@ -255,20 +346,33 @@ public void testOptionsBelowEE9() {
...
-                        .withKind(SpanKind.CLIENT)
-                        .withAttribute(SemanticAttributes.HTTP_METHOD, "OPTIONS")
-                        .withAttribute(SemanticAttributes.HTTP_STATUS_CODE, 200L)
-                        .withAttribute(SemanticAttributes.HTTP_URL, testUri.toString()));
-
-        assertThat(serverSpan, isSpan()
-                        .withKind(SpanKind.SERVER)
-                        .withAttribute(SemanticAttributes.HTTP_METHOD, "OPTIONS")
-                        .withAttribute(SemanticAttributes.HTTP_STATUS_CODE, 200L));
+if(featureVersion.equals("2.0")){
```
Comment (benjamin-confino) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/jaxrspropagation/methods/JaxRsMethodTestServlet.java:349]: Indentation.

File: dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/jaxrspropagation/methods/JaxRsMethodTestServlet.java [hunk 2]
```diff
@@ -287,16 +391,29 @@ public void testOptionsAboveEE8() {
...
```
Comment (benjamin-confino) [inline dev/io.openliberty.microprofile.telemetry.1.0.internal_fat/fat/src/io/openliberty/microprofile/telemetry/internal_fat/apps/jaxrspropagation/methods/JaxRsMethodTestServlet.java:394]: Indentation

**Review by benjamin-confino — APPROVED** [review]
I think this is good to go. However in a follow up I think this would benefit from commentary: 

`// MP Telemetry 2.0 does not consume io.opentelemetry.semconv.trace.attributes.SemanticAttributes` could be something like. "In MP Telemetry 2.0 SemanticAttributes was moved to a new package, so we use import static to allow both versions to coexist"

```
    /*
     * Ideal behaviour for this test would be HHTP_ROUTE = /getSubResourceWithQueryParam/details
     * Due to the current behaviour when sub resources are used is that only the context root is returned
     */
```
Was not clear to me.

And when you do skip repeats it would be good to ensure each one has a comment saying why its skipped on those versions.

**Pull request merged**